### PR TITLE
Update ThingProperties.h

### PR DESCRIPTION
**Pull request by etarabey**
Added accelerometer update function.

### CODE REVIEW

**Comment by jeremycote** [comment]
Could you add the rest of your changes to this PR? That way, it’s all together in one commit.

**Comment by jeremycote** [comment]
Oh, nvm, I see you have a second PR

**Comment by etarabey** [comment]
So, add it to my forked repo?

File: include/ThingProperties.h [hunk 1]
```diff
@@ -28,6 +28,12 @@ void updateBatteryVoltage(float voltage) {
     battery_Voltage = voltage;
 }
 
+void updateAcceleration(accel_t acceleration_struct){ 
+    new_accel_x = acceleration_struct.accel_x;
```
Comment (jeremycote) [inline include/ThingProperties.h:32]: I don’t see new_accel_ variables in this file. They should be with the battery_voltage variable.

**Pull request closed without merging**